### ISSUE TITLE
Compiler fails for generic tables

### ISSUE THREAD

**Issue by PtiPingouin** [gfx/Android-Orma#265]
Any generic table,
ie:
```
@Table
class A extends B<String> {
    ...
}
```
will cause the compiler to fail (even if the parameter is never used), with:

```
Error:[OrmaProcessor] Unexpected exception while processing annotations
  	at com.github.gfx.android.orma.processor.model.SchemaDefinition.collectColumns(SchemaDefinition.java:191)
  java.lang.NullPointerException
  	at com.github.gfx.android.orma.processor.model.SchemaDefinition.collectColumns(SchemaDefinition.java:188)
  	at com.github.gfx.android.orma.processor.model.SchemaDefinition.<init>(SchemaDefinition.java:94)
  	at com.github.gfx.android.orma.processor.OrmaProcessor.lambda$buildTableSchemas$7(OrmaProcessor.java:137)
```

**Comment by gfx** [comment]
`NullPointerException` is a bug, but I'm not sure parameterized tables are valid. Will look into it.

**Comment by gfx** [comment]
Fixed in #266, but you can't use `T` as a column type, though.

**Comment by PtiPingouin** [comment]
Arigato!
